File: Assignment_39/Program2.cpp
#include<iostream>
using namespace std;

template<class T>
T Frequency(T *Arr, int iSize,T no)
{
    int i = 0,iCount = 0;
    for(i = 0; i < iSize; i++)
    {
        if(Arr[i] == no)
        {
            iCount++;
        }
    }
    return iCount;  
}

int main()
{
    int iRet = 0;

    int arr[] = {10,20,30,40,10,20,30,10,30};
    double brr[] = {10.3,20.2,30.4,40.3,10.7,20.5,30.4,10.6,30.7};

    iRet = Frequency(arr,9,10);
    cout<<"Frequency is : "<<iRet<<"\n";

    iRet = Frequency(brr,9,10.3);
    cout<<"Frequency is : "<<iRet<<"\n";

    return 0;
}

File: Assignment_44/Program2.java
import java.util.*;

class Matrix
{private int Arr[][];

    public Matrix(int A, int B)
    {
        Arr = new int[A][B];
    }

    public void Accept()
    {
        System.out.println("Please enter elements of matrix");

        Scanner sobj = new Scanner(System.in);

        int i = 0,j = 0;

        for(i = 0; i < Arr.length; i++)
        {
            for(j = 0; j < Arr[i].length; j++)
            {
                Arr[i][j] = sobj.nextInt();
            }
        }
    }

    public int Frequency(int Row,int Col,int iNo)
    {
        int i = 0,j = 0,iCount = 0;

        for(i = 0; i < Arr.length; i++)
        {
            for(j = 0; j < Arr[i].length; j++)
            {
                if(Arr[i][j] == iNo)
                {
                    iCount++;
                }
            }
        }
        return iCount;
    }
}

public class Program2
{
    public static void main(String A[])
    {
        int i = 0,j =0;
        int iRet = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the Number of Rows : ");
        int Row = sobj.nextInt();

        System.out.println("Enter the Number of Columns : ");
        int Col = sobj.nextInt();

        System.out.println("Enter the Number : ");
        int iNo = sobj.nextInt();

        Matrix mobj = new Matrix(Row, Col);

        mobj.Accept();

        iRet = mobj.Frequency( Row, Col,iNo);
        System.out.println("Frequency is : "+iRet);

    }


    
}